src/nccl_ofi_api.c

```diff
@@ -234,7 +234,7 @@ ncclResult_t nccl_net_ofi_connect(int dev_id, void *handle, void **sComm)
 			return ncclInternalError;
 		}
 
-		int ret = device->get_ep(device, &base_ep);
+		int ret = device->get_ep(device, &base_ep, true);
```

Comment:
You should not be blindly passing true here; this is a sign you have the API wrong.